roles/nv_gpu_install_from_commit/files/operator_helper_image_builder.yml

```diff
@@ -14,47 +14,68 @@ spec:
       # that runs safely with privileges within the container.
+      # The containers created by this image also come with a
+      # Buildah development environment in /root/buildah.
       #
-      # --> adapted to use podman instead of buildah
-      FROM quay.io/fedora/fedora:32-x86_64
-
-      # Don't include container-selinux and remove
-      # directories used by dnf that are just taking
-      # up space.
-      RUN yum -y install podman fuse-overlayfs; rm -rf /var/cache /var/log/dnf* /var/log/yum.*
-
+      FROM registry.fedoraproject.org/fedora:latest
+      ENV GOPATH=/root/buildah
+      
+      # Install the software required to build Buildah.
+      # Then create a directory and clone from the Buildah
+      # GitHub repository, make and install Buildah
+      # to the container.
+      # Finally remove the buildah directory and a few other packages
+      # that are needed for building but not running Buildah
+      RUN useradd build; yum -y update; yum -y reinstall shadow-utils; yum -y install --enablerepo=updates-testing \
+           make \
+           golang \
+           bats \
+           btrfs-progs-devel \
+           device-mapper-devel \
+           glib2-devel \
+           gpgme-devel \
+           libassuan-devel \
+           libseccomp-devel \
+           git \
+           bzip2 \
+           xz \
+           go-md2man \
+           runc \
+           fuse-overlayfs \
+           fuse3 \
+           containers-common; \
+           mkdir /root/buildah; \
+           git clone https://github.com/containers/buildah /root/buildah/src/github.com/containers/buildah; \
+           cd /root/buildah/src/github.com/containers/buildah; \
+           make;\
+           make install;\
+           rm -rf /root/buildah/*; \
+           yum -y remove bats golang go-md2man; \
+           yum clean all;
+      
+      ADD https://raw.githubusercontent.com/containers/buildah/master/contrib/buildahimage/stable/containers.conf /etc/containers/
+      
       # Adjust storage.conf to enable Fuse storage.
-      RUN sed -i 's|# Storage options to be passed to underlying storage drivers|# Storage options to be passed to underlying storage drivers\nmount_program = "/usr/bin/fuse-overlayfs"|' /etc/containers/storage.conf
-
-      RUN sed -i 's|additionalimagestores = \[|additionalimagestores = \[\n  "/var/lib/shared"|' /etc/containers/storage.conf
-
-      RUN [ -c /dev/fuse ] || mknod /dev/fuse c 10 229
-
-      RUN mkdir -p /var/lib/shared/overlay-images /var/lib/shared/overlay-layers; touch /var/lib/shared/overlay-images/images.lock; touch /var/lib/shared/overlay-layers/layers.lock
-
-      # Set up environment variables to note that this is
-      # not starting with user namespace and default to
-      # isolate the filesystem with chroot.
-      ENV _BUILDAH_STARTED_IN_USERNS="" BUILDAH_ISOLATION=chroot
-
-      # GPU-operator-CI specific
-
-      RUN yum -y install git make
```

Comment:
just a nit, but can you keep that at the bottom of the file, so that we track explicitly the dependencies specific to the building the GPU operator?

Reply:
Not needed look at:

https://github.com/openshift-psap/ci-artifacts/blob/e5fe7babfb6bf955a3e7cca00751cf540fee0ca3/roles/nv_gpu_install_from_commit/files/operator_helper_image_builder.yml#L38

Reply:
hum, maybe then just add a comment:
```
# Packages required for building the GPU-operator-CI image:
# - git 
# - make
```